plugins/woocommerce-admin/client/products/sections/pricing-section.tsx

```diff
@@ -128,6 +128,7 @@ export const PricingSection: React.FC = () => {
 					>
 						<InputControl
 							{ ...regularPriceProps }
+							className="woocommerce-field-width"
```

Comment:
I'd put this in `currencyInputProps` above. That way, it will get merged I believe, and not just override any other classes already set on the component via `regularPriceProps` or anything else up the chain.

Reply:
Ah, wait... we are wrapping the `InputControl` in a `BaseControl`, and we are pulling out `className` to set on the `BaseControl`. So, I'd still suggest putting it in `currencyInputProps`, but only set it on either `BaseControl` or `InputControl`... not sure which one would be best, probably `InputControl`, since the `label` or `help` could be longer (as your screenshot shows).

Honestly, with what we are doing here with `BaseControl` and `InputControl`, we ideally should have a new component such as `CurrencyInput` that takes care of all of this. Otherwise, things are quickly going to get messy and difficult to maintain.